src/gestures.cc

```diff
@@ -1,353 +0,0 @@
-/* This file is (c) 2014 Abs62
- * Part of GoldenDict. Licensed under GPLv3 or later, see the LICENSE file */
-
-#include <QTouchEvent>
-#include <QSwipeGesture>
-#include <QVariant>
-#include <math.h>
-#include "ui/articleview.hh"
-#include "gestures.hh"
-
-namespace Gestures {
-
-Qt::GestureType GDPinchGestureType;
-Qt::GestureType GDSwipeGestureType;
-static bool fewTouchPointsPresented;
-
-bool isFewTouchPointsPresented()
-{
-  return fewTouchPointsPresented;
-}
-
-QSwipeGesture::SwipeDirection getHorizontalDirection( qreal angle )
-{
-  if ( ( 0 <= angle && angle <= 45 ) || ( 315 <= angle && angle <= 360 ) )
-    return QSwipeGesture::Right;
-
-  if ( 135 <= angle && angle <= 225 )
-    return QSwipeGesture::Left;
-
-  return QSwipeGesture::NoDirection;
-}
```

Comment:
I don't know why he's calculating angles. But
```
0 <= angle && angle <= 45 
```
This thing is simply `y/x < 1` if `(x,y)` is a point to the origin

I set the ratio to 6, so that moving 2 figures on the x-axis 6 times more than the y-axis triggers forward/backward